### ISSUE TITLE
Perform Accessibility Audit of SimpleReport Web Application and Static Site

### ISSUE THREAD

**Issue by jayna-SkylightDigital** [CDCgov/prime-simplereport#5534]
### Background / Description
In July 2022, the CDC Accessibility team reached out to SimpleReport via Index Analytics (Support) to alert us of a possible violation of Section 508 of the Rehabilitation Act. Our team was required to take immediate action to remedy the situation, performing an audit on the entire site and providing evidence that the offending issues had been addressed. Since that time, the SimpleReport team has performed quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

The **PRIME Accessibility Best Practices** can be found here: [https://cdc.sharepoint.com/:w:/r/teams/USDSatCDC/_layouts/15/Doc.aspx?sourcedoc=%7B31480032-AB2F-4106-A510-6AFA860003F7%7D&file=PRIME%20Accessibility%20Best%20Practices.docx&action=default&mobileredirect=true&DefaultItemOpen=1]

### Requirements
1. Perform an audit of the SimpleReport Web Application using Deque's axe DevTools extension.
2. Perform an audit of the SimpleReport static website using Deque's axe DevTools extension.
3. Generate Github tickets based on the findings from the audit(s)

### Acceptance Criteria / Definition of Done
- Create tickets to complete the accessibility audit of all pages on the SimpleReport app and static site
- Tickets based on the findings from the audit(s) have been added to the backlog and assigned to the Epic **Q3-FY23 Perform Quarterly Accessibility Audit (G&E)** #5493

**Comment by mehansen** [comment]
Break up the accessibility audit work into sub tickets.

List of pages that need the audit:
https://docs.google.com/spreadsheets/d/1m7uqLc-bMU9NRLMG5vTBXp1xPpoNqhBKw-V1UodDHQ4/edit#gid=114781190


Header & user menu drop-down
Support admin
Dashboard
Conduct tests
	- inc test questionnnaire
	- patient search
View test results
	- result details modal
	- download test results modal
	- print/email/text results modals
Upload spreadsheet
View upload history
Bulk results upload guide (link from upload spreadsheets page)
View submission details (from upload history page)
Patients
	- archive patient modal
	- patient search
Add/edit patient
Manage users
	- user info
	- facility access
Manage facilities
Add/edit facility
Manage organization
Patient self-registration

what are other links?
patient self-registration link
link sent to patient to view their results

Terms of service for patient registration link
Register for your test from patient registration link

**Comment by mehansen** [comment]
Closing this as complete as we have created multiple tickets for individual pieces of the work, all linked to the epic #5493 in ZenHub
